dist/index.html

```diff
@@ -8,7 +8,6 @@
 <script defer src="main.js"></script></head>
 <body>
     <h1>Leaderboard</h1>
-    <div id="content">
-    </div>    
```

Comment:
🌸  Hey! This is great. Since this is the only section on the page you could us ``main`` tag instead of div for better HTML semantics